lib/bundle/brew_dumper.rb

```diff
@@ -16,12 +16,18 @@ def self.formulae
     end
 
     def self.dump
+      bs = BrewServices.new
       formulae.map do |f|
+        if bs.started?(f[:full_name])
+          restart = ', service_restart: true'
+        else
+          restart = ''
+        end
         if f[:args].empty?
-          "brew '#{f[:full_name]}'"
+          "brew '#{f[:full_name]}'" + restart
         else
           args = f[:args].map { |arg| "'#{arg}'" }.sort.join(", ")
-          "brew '#{f[:full_name]}', args: [#{args}]"
+          "brew '#{f[:full_name]}', args: [#{args}]" + restart
```

Comment:
It may also make sense to split up building the string up with `+=` and build first the `brew` with `full_name`, then, `unless f[:args].empty?` add `args` and `unless f[:service_started].empty?` to avoid duplication.